src/test/ui/resolve/issue-81508.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0433]: failed to resolve: use of undeclared type `Foo`
+  --> $DIR/issue-81508.rs:15:24
+   |
+LL |         println!("{}", Foo::Bar);
+   |                        ^^^-----
+   |                        |
+   |                        use of undeclared type `Foo`
+   |                        help: try using the similarly named variable instead: `Foo`
```

Comment:
I think this message is confusing, because it seems to suggest using `Foo`, which the user is already doing. I think we should instead point to the declaration of `Foo` and say something like:
```
`Foo` is defined here, but is not a type
```

Reply:
@kwj2104: yes, that looks like the sort of thing I had in mind!

Reply:
New format for errors now points to declaration line: 

```
error[E0433]: failed to resolve: use of undeclared type `Baz`
  --> $DIR/issue-81508.rs:11:20
   |
LL |     let Baz: &str = "";
   |         --- help: Baz is defined here, but is not a type
LL | 
LL |     println!("{}", Baz::Bar);
   |                    ^^^ use of undeclared type `Baz`
```